OctaneVSPlugin/ViewModel/OctaneMyItemsViewModel.cs

```diff
@@ -19,6 +19,7 @@
 using MicroFocus.Adm.Octane.VisualStudio.View;
 using Microsoft.VisualStudio.PlatformUI;
 using System;
+using System.Linq;
```

Comment:
needed?

Reply:
yes, Linq is needed to apply stream operations on collections